linera-service/src/cli_wrappers.rs

```diff
@@ -94,16 +97,11 @@ pub struct ClientWrapper {
 impl ClientWrapper {
     fn new(
         tmp_dir: Rc<TempDir>,
-        database: Database,
         network: Network,
         testing_prng_seed: Option<u64>,
         id: usize,
     ) -> Self {
-        let storage = match database {
-            Database::RocksDb => format!("rocksdb:{}/client_{}.db", tmp_dir.path().display(), id),
-            Database::DynamoDb => format!("dynamodb:client_{}.db:localstack", id),
-            Database::ScyllaDb => format!("scylladb:table_client_{}_db", id),
-        };
+        let storage = format!("rocksdb:{}/client_{}.db", tmp_dir.path().display(), id);
```

Comment:
Make sense but the feature `rocksdb` is now mandatory. Isn't it?

Reply:
Absolutely, this is now the convention. We put the wallet on `RocksDb`.